packages/EasyCore/src/Search/ElasticsearchSearchService.php

```diff
@@ -61,4 +62,26 @@ public function search(
             'body' => $body,
         ]);
     }
+
+    /**
```

Comment:
Let's add a description: 
Needed to avoid "[match_all] query malformed, no start_object after query name" error.

Reply:
Done.